setup.cfg

```diff
@@ -117,6 +117,10 @@ typecheck =
     types-setuptools==57.4.0
     types-toml==0.10.0
 
+[options.package_data]
+yt =
+    py.typed
```

Comment:
I'm surprised this section isn't already present. Maybe I should add this file to MANIFEST.in instead ?

Reply:
Answered my own question thanks to https://stackoverflow.com/questions/45147837/including-data-files-with-setup-py

We may be able to migrate away from MANIFEST.in and put all that stuff in setup.cfg in the future, but that's out of scope here, and I think it's much preferable to keep everything in one place as long as possible. I'll use the MANIFEST instead.